### ISSUE TITLE
[BUG] AutoBlock doesn't work at high cps and AutoBlock is not 100% effective under any settings.

### ISSUE THREAD

**Issue by guosic** [CCBlueX/LiquidBounce#2660]
### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

latest git-1411acc

### Operating System

Windows 11

### Minecraft Version

1.20.4

### Describe the bug

Observing the blocking status on the test server for a period of time.

https://github.com/CCBlueX/LiquidBounce/assets/101028480/8f198b82-5895-46f1-a601-760dce7294ea

### Steps to reproduce

-

### Client Log

```shell
-
```

**Comment by rreal11** [comment]
hold right click

**Comment by larryngton** [comment]
use basic instead of interact ab

**Comment by guosic** [comment]
> hold right click

Yes, it's currently possible to manually hold down the right click to do a 100% block but not automatically.

**Comment by guosic** [comment]
> use basic instead of interact ab

That’s not the issue.
